### DISCUSSION TITLE
Gracias

### DISCUSSION THREAD

**Discussion by Ynaso** [cacao-accounting/cacao-accounting#39] [Show and tell]
Hola, soy de Nicaragua, y en vista de ayudarle a uno de mis suegros a poder llevar una contabilidad automatizada busque un programa contable de codigo abierto, gracias por tu aporte se programar en python pero no se como podria ayudar a este proyecto si es es que se puedeo ;=)

**Comment by williamjmorenor** [comment]
> Hola, soy de Nicaragua, y en vista de ayudarle a uno de mis suegros a poder llevar una contabilidad automatizada busque un programa contable de codigo abierto, gracias por tu aporte se programar en python pero no se como podria ayudar a este proyecto si es es que se puedeo ;=)

Hola que bien que seas de nicaragua, que bueno que encontraste el proyecto.

Había dejado un poco de lado esto, pero lo estoy retomando, espero pronto poder tener al menos lo mínimo para ingresar comprobantes contables y generar los reportes básicos.
